lib/reporters/xunit.js

```diff
@@ -130,7 +130,7 @@ XUnit.prototype.test = function(test) {
 
   if (test.state === 'failed') {
     var err = test.err;
-    this.write(tag('testcase', attrs, false, tag('failure', {}, false, cdata(escape(err.message) + '\n' + err.stack))));
```

Comment:
Sorry, I'm don't know much about this topic. I don't see:
- Why (before your changes) the data was being escaped even though it was being put inside a `CDATA` tag. (which afaik is used to avoid having to escape chars)
- Why escaping the data is preferred to just putting the data inside a `CDATA` tag. (since using the `CDATA` tag uses less bytes and makes the data readable by a human on a quick glance)